docs/src/usage/git-hooks-pre-commit.md

```diff
@@ -0,0 +1,10 @@
+To integrate selene with git, you can use [pre-commit](https://pre-commit.com). Once you have [pre-commit installed](https://pre-commit.com/#install), add this to the `.pre-commit-config.yaml` at root of your repository:
+```yaml
+repos:
+  - repo: https://github.com/Kampfkarren/selene
+    rev: '' # Change this value to use a different version of Selene
```

Comment:
Does this do anything with `selene-system`, since it uses whatever is installed locally? If not then I think it'd be less confusing to separate the system and docker examples.

Reply:
Separated out each type of hook into it's own piece of documentation in [5f1a755](https://github.com/Kampfkarren/selene/pull/565/commits/5f1a755b32acc8d36ed96c417eabaf81159018ee)